pandas/core/groupby/numba_.py

```diff
@@ -0,0 +1,172 @@
+"""Common utilities for Numba operations with groupby ops"""
```

Comment:
Why `numba_` and not `_numba`? I would expect this module would be private and expose no external API.

Reply:
the entire pandas/core tree is private